### ISSUE TITLE
Resource order not applied with `syncDataOnLoad`

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#6039]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=44&t=23654&p=117147#p117147)

Using the [React state demo](https://bryntum.com/products/scheduler/examples/frameworks/react/javascript/react-state/build/), change the second dataset to match the first, but in a different order, like this 

```
...
resources: [
        [
            { id: 'r1', name: 'Mike' },
            { id: 'r2', name: 'Linda' },
            { id: 'r3', name: 'Chang' },
            { id: 'r4', name: 'Kate' },
            { id: 'r5', name: 'Lisa' },
            { id: 'r6', name: 'Steve' },
            { id: 'r7', name: 'Mark' },
            { id: 'r8', name: 'Madison' },
            { id: 'r9', name: 'Hitomi' },
            { id: 'r10', name: 'Dan' }
        ],
        [
            { id: "r6", name: "Steve" },
            { id: "r7", name: "Mark" },
            { id: "r8", name: "Madison" },
            { id: "r9", name: "Hitomi" },
            { id: "r10", name: "Dan" },
            { id: "r1", name: "Mike" },
            { id: "r2", name: "Linda" },
            { id: "r3", name: "Chang" },
            { id: "r4", name: "Kate" },
            { id: "r5", name: "Lisa" },
        ],
...
```
The expected behavior is that the resources order should change, but as you'll see in the video, that's not the current behavior.

https://user-images.githubusercontent.com/16693227/214376051-c516448a-d2c5-4d95-80d9-cf8e84dce146.mp4

**Comment by isglass** [comment]
Most likely this is because wrappers use syncDataOnLoad by default, bug probably being it not changing order of records when applying changes. Makes sense that it should.

As a workaround of sorts, disable syncDataOnLoad on the resource store

**Comment by shivamchamoli09** [comment]
Hi team, is this resolved in the latest release?

**Comment by matsbryntse** [comment]
Yes, fixed in 5.2.10
